### ISSUE TITLE
Error with the new file when creating a new stub database

### ISSUE THREAD

**Issue by quimmol** [a1ext/labeless#59]
#### IDA-side information 

| Question                      | Answer
|-------------------------------|--------------------
| IDA-side OS version           | Windows 10
| IDA PRO bitness (not IDB/I64) | 64
| IDA plugin bitness            | 32
| IDA plugin version            | v1.1.2.85

#### Debugger-side information

| Question                         | Answer
|----------------------------------|--------------------
| OS version (client)              | Windows 10
| Plugin version                   | 1.1.2.85
| `labeless` python module version | 1.1.3.0
| Debugger type                    | x64dbg-x32, x64dbg-x64

### Problem Description
The bug occurs when loading a stub database on IDA 7.6 or 7.5, don't appear the Load a new file dialog and directly IDA will fail to load the new file.

### Steps to Reproduce
1. Open IDA 7.6/7.5
2. Load stub database
3. Create Sample stub on any location
4. Error from IDA opening the file.

### Complementary information

![labeless](https://user-images.githubusercontent.com/8196326/112405762-351e3800-8d13-11eb-854b-434f7fd30efa.gif)

**Comment by a1ext** [comment]
Hi @quimmol, it seems like plugin failed to write a stub exe to that folder. Does IDA have permissions to write there?

**Comment by quimmol** [comment]
Yes, i tried to create the stub database on common locations that i use with IDA, also with admin priv. and the error is still the same.

This has more to do with IDA than labeless i understand but if someone has encountered this error so it can be fixed.

Sorry for the late response.

**Comment by a1ext** [comment]
I will try to reproduce it. Thanks

**Comment by a1ext** [comment]
Maybe your antivirus blocked the PE stub the plugin drop to allow IDA load empty sample... Have you seen any warning from your AV?
